content/posts/newsletter/2021-02-05.md

```diff
@@ -0,0 +1,239 @@
+date: 2021-01-05 18:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter covering the month of December 2020 and January 2021.
+
+Happy to have you back reading! We hope this year will be a great one you, but as well for XMPP!
+
+In December 2020 we discussed the last year of producing and publishing of the XMPP Newsletter, [read the notes here](https://github.com/xsf/xmpp.org/pull/851#issuecomment-754919811)!
+
+Many projects and their efforts in the XMPP community are a result of people’s voluntary work. If you are happy with the services and software you may be using, especially throughout the last year, please consider to say thanks or help these projects!
+
+### Newsletter translations
+
+Translations of the XMPP Newsletter will be released here:
+
+- French on [linuxfr.org](https://linuxfr.org/tags/xmpp/public) and [jabberfr.org](https://news/jabberfr.org/category/newsletter/).
+
+## XSF Announcement
+
+Regarding the announcement by Let's Encrypt to switch away from their Root CA certificate: Now they are extending their Android device compatibility of their certificates](https://letsencrypt.org/2020/12/21/extending-android-compatibility.html). [Read their the original post](https://letsencrypt.org/2020/11/06/own-two-feet.html). 
+
+Edward Maurer wrote a commentary blog post regarding on the current situation of instant messaging communication and XMPP: [Instant Messaging: It's not about the app](https://xmpp.org/2021/01/instant-messaging-its-not-about-the-app/). There are also translations in [German](https://xmpp.org/2021/01/instant-messaging-es-geht-nicht-um-die-app/), [French](https://xmpp.org/2021/01/messagerie-instantanee-il-ne-sagit-pas-de-lapplication/), [Spanish](https://xmpp.org/2021/01/mensajeria-instantanea-no-se-trata-de-la-aplicacion/) and [Romanian](https://xmpp.org/2021/01/mesagerie-instantanee-nu-este-vorba-despre-aplicatie/). Thanks to all contributors!
+
+## Events
+
+[Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): Monthly Meeting of XMPP Enthusiasts in Berlin - always 2nd Wednesday of
+every month
+
+## Articles 
+
+The developers of the Prosody XMPP server wrote  [a recap post of 2020](https://blog.prosody.im/2020-retrospective/) summarizing what happened to the project in the past year. The post contains some nice statistics about how many Prosody deployments are running around the world.
+
+Wolthera wrote another article explaining how to write an [XMPP Client in Godot](https://wolthera.info/2020/12/xmpp-client-in-godot/).
+
+[Sponsoring Group Chat Encryption in Siskin](https://snikket.org/blog/sponsoring-group-omemo-in-siskin/): For some time XMPP has been in need of a good solution for Apple’s mobile operating system, iOS. In 2021 we now have a number of active projects. Thanks to a private donor, Snikket Chat was able to sponsor the development of end-to-end encrypted group chats (OMEMO) in Siskin, the iOS XMPP client!
```

Comment:
This post is from **November**, why is it here?

Reply:
Because it hasn’t been mentioned before, sadly.

Reply:
It was mentioned in the [November newsletter](https://xmpp.org/2020/11/newsletter-11-november/).

Reply:
Darn, you are right. That paragraph has been added because Siskin had its 6.2 release (which features the announced changes), but that gets mentioned elsewhere in this issue :)

Reply:
Yes, I will take it out - thanks! Actually I plan to crosscheck but forgot